addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -7,15 +7,16 @@ class FleetVehicleLogServices(models.Model):
     _inherit = 'fleet.vehicle.log.services'
 
     account_move_line_id = fields.Many2one(comodel_name='account.move.line')
-    account_move_state = fields.Selection(related='account_move_line_id.parent_state', store=True)
-    amount = fields.Monetary('Cost', compute="_compute_amount", inverse="_inverse_amount", readonly=False, store=True, tracking=True)
-    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', compute="_compute_vehicle_id", store=True, readonly=False, required=True)
+    account_move_state = fields.Selection(related='account_move_line_id.parent_state')
+    amount = fields.Monetary('Cost', compute="_compute_amount", inverse="_inverse_amount",
+        readonly=False, store=True, tracking=True)
+    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', compute="_compute_vehicle_id",
+        store=True, readonly=False, required=True)
```

Comment:
not sure if this diff is necessary.
we usually put all args on new lines, except for `compute`, `inverse`, `store`, `readonly`
feel free to resolve :)

Reply:
I made the change to avoid having lines longer than 120 characters :) What would be your suggestion? Should I keep it as it was?

Reply:
TBH I have a horizontal scroll on my mouse, so I don't mind the longer lines 🤣 so I would probably only make the diff if I was changing something else
but if you want to shorten it, this would be my suggestion:
```suggestion
    amount = fields.Monetary(
        string="Cost",
        compute='_compute_amount', inverse='_inverse_amount', readonly=False, store=True,
        tracking=True)
    vehicle_id = fields.Many2one(
        comodel_name='fleet.vehicle',
        string="Vehicle",
        compute='_compute_vehicle_id', store=True, readonly=False,
        required=True)
```

Reply:
As we (sadly) have no standard on that so I'll keep the change this way, but I'm all down for expliciting string and comodel_name :raised_hands: